Mathlib/Tactic/FindSyntax.lean

```diff
@@ -75,12 +75,20 @@ elab "#find_syntax " id:str d:(&" approx")? : command => do
       if !declName.isInternal && !ls.isEmpty then symbs := symbs.insert (declName, ls)
   -- From among the parsers in `symbs`, we extract the ones whose `symbols` contain the input `str`
   let mut msgs := #[]
+  let env ← getEnv
+  let allModules := env.allImportedModuleNames
   for (nm, ar) in symbs.toList do
     let rem : String := " _ ".intercalate (ar.map litToString).toList
     -- If either the name of the parser or the regenerated syntax stub contains the input string,
     -- then we include an entry into the final message.
     if 2 ≤ (nm.toString.splitOn id.getString).length || 2 ≤ (rem.splitOn id.getString).length then
-      msgs := msgs.push <| .ofConstName nm ++ m!":\n  '{rem.trim}'\n"
+      let some mod := env.getModuleIdxFor? nm | pure default
```

Comment:
If you're submitting a docs PR: documenting what this does could also be nice (just relating it to const2ModIdx would do).

Reply:
I forgot to comment here: `getModuleIdxFor?` is contained in a wall of declarations none of which has a doc-string.

Besides, the variable names for `getModuleIdxFor?` are pretty clear: `env`(inronment) and `declName` and it returns a `ModuleIdx`.

Reply:
I know you got some direction that module indexes might be public API, but `Lean.findModuleOf?` seems a lot better to me.